.azure/pipelines/ci.yml

```diff
@@ -577,9 +576,16 @@ stages:
       agentOs: Windows
       timeoutInMinutes: 180
       steps:
-      - script: .\restore.cmd -ci
-        displayName: Restore
-      - script: .\build.cmd -ci -NoRestore -test -projects eng\helix\helix.proj /p:IsRequiredCheck=true /p:IsHelixJob=true /p:BuildAllProjects=true /p:BuildNative=true /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log -bl
+      # Build the shared framework
+      #- script: ./build.cmd -ci -all -pack -arch x64 /bl:artifacts/log/helix.build.x64.binlog
+      #  displayName: Build shared fx
+      # Build the x86 shared framework
+      # This is going to actually build x86 native assets.
+      #- script: ./build.cmd -ci -arch x86 -pack -all -buildNative -noBuildJava /p:OnlyPackPlatformSpecificPackages=true /bl:artifacts/log/helix.build.x86.binlog $(_BuildArgs)
+      #  displayName: Build x86 shared framework pieces
+      - script: ./src/ProjectTemplates/build.cmd -ci -pack -NoRestore -NoBuilddeps "/p:RunTemplateTests=true /bl:artifacts/log/helix.template.pack.binlog"
```

Comment:
This line is missing the property `/p:ASPNETCORE_TEST_LOG_DIR=artifacts/log`. We need to set this property at build time not test time.